test/extended/operators/tolerations.go

```diff
@@ -0,0 +1,93 @@
+package operators
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = Describe("[Feature:Platform][Smoke] Managed cluster should", func() {
+	oc := exutil.NewCLIWithoutNamespace("operators")
+
+	It("ensure control plane operators do not make themselves unevictable", func() {
```

Comment:
perhaps we should talk about tolerations as the code below tries to find tolerations.